src/applications/yellow-ribbon/manifest.json

```diff
@@ -0,0 +1,13 @@
+{
+  "appName": "Yellow Ribbon Schools",
+  "entryFile": "./yellow-ribbon-entry",
+  "entryName": "yellow-ribbon",
+  "rootUrl": "/yellow-ribbon/schools",
+  "production": false,
+  "template": {
+    "vagovprod": false,
+    "vagovstaging": false,
+    "vagovdev": false,
+    "layout": "page-react.html"
+  }
+}
```

Comment:
This is a full-page application instead of a widget living on a Drupal page, so this `manifest.json` will create the page URL off the `rootUrl` and `template` property.